app/decorators/case_extend_sar_deadline_decorator.rb

```diff
@@ -8,4 +8,8 @@ class CaseExtendSARDeadlineDecorator < Draper::Decorator
   def allow_extension_period_selection?
     !object.deadline_extended? && object.deadline_extendable?
   end
+
+  def time_unit_string(time_period)
+    object.deadline_calculator.time_units_desc_for_deadline(time_period > 1)
```

Comment:
We can probably use the built-in Rails pluralize helper instead of writing our own method to pluralize 'day' -> 'days'

Reply:
thank you will change to that